website/notifications/emails.py

```diff
@@ -63,7 +63,7 @@ def email_digest(recipient_ids, uid, event, user, node, timestamp, **context):
         context['localized_timestamp'] = localize_timestamp(timestamp, recipient)
         message = mails.render_message(template, **context)
 
-        if user._id != recipient._id:
+        if message and user._id != recipient._id:
```

Comment:
I honestly can't think of a reason why this would be an empty string, but suspect that is what happened when @abought received an email digest with no messages in it.